package-scope.qmd

```diff
@@ -1,55 +1,43 @@
 ---
-title: "Package scope"
+title: "Plotting functionality"
 ---
 
-## Visualisation in R packages
-
-The use of visualisation is a fundamental step in scientific analysis and a [core component of data science](https://r4ds.had.co.nz/introduction.html). However, the question of whether a package should include plotting functionality is not obvious.
-
-::: {.callout-note title="Not all package plotting requires are equal"}
-
-This section details the Epiverse-TRACE reasoning for the inclusion or exclusion of functionality around plotting. It is not a commentary generalisation to all R packages.
+Visualisation is key to (data) science and in this section we detail how we approach plotting functionality for our Epiverse packages.
 
+:::{.callout-warning}
+There is ongoing work around theming of plots. Please see [{epiversetheme}](https://epiverse-trace.github.io/epiversetheme/) for more information on this.
 :::
 
-## Epiverse-TRACE philosophy
-
-The Epiverse-TRACE philosophy is to only include minimal, dependency-free, plotting functionality, and to instead provide examples and templates of plotting using {ggplot2} in vignettes. An example of the inclusion of a plotting function is the [`<epidist>` plotting method](https://epiverse-trace.github.io/epiparameter/reference/plot.epidist.html) in {epiparameter}.
+## Approach
 
-Epiverse-TRACE packages aim to provide comprehensive documentation that explains exported functions and demonstrates their applications, both through [{roxygen2}](https://roxygen2.r-lib.org/) function-specific documentation, or via longer-form vignettes. It is in the vignettes that Epiverse-TRACE packages show examples of how the data or model outputs can be visualised, with the associated code chunks required to produce the plot.
+Our approach to plotting functionality is twofold:
 
-It is with this provision of plotting code chunks we hope to provide the community with a resource from which their desired plots can be produced. Vignettes use the {ggplot2} package to provide easily extensible plots with the addition of [layers](https://rpubs.com/hadley/ggplot2-layers).
+1. We minimize dependencies used for plotting functions in our packages.
```

Comment:
I think this should be reframed as: minimise lesser used packages, or plotting packages with system dependencies. In my opinion {ggplot2} is not particularly lightweight (as outlined below), so it doesn't really match a minimal dependency philosophy. Rather because it is so ubiquitous it unlikely that many users will need to install it when using an Epiverse package with {ggplot2} as a dependency. This argument could be extended to gg-derivative packages as their dependency stack will likely highly overlap with {ggplot2}, making them good candidates as dependencies as well.

Reply:
Thanks! I agree and that is perfectly in line [with minimizing dependencies as in our existing standards](https://epiverse-trace.github.io/blueprints/dependencies.html#minimizing-dependencies-and-prioritizing-stability). 

I made sure to cross-link to that so that we can have consistency and not rearticulate standards in multiple places.